stingray/gti.py

```diff
@@ -194,8 +194,8 @@ def cross_two_gtis(gti0, gti1):
     cross_gtis : From multiple GTI lists, extract common intervals *EXACTLY*
 
     """
-    gti0 = np.array(gti0, dtype=np.longdouble)
-    gti1 = np.array(gti1, dtype=np.longdouble)
+    gti0 = np.asarray(gti0)
```

Comment:
Agreed. I think users need to work out their own precision. Can't do *all* the work for them. :)